components/transaction/internal/ports/http/assetrate.go

```diff
@@ -0,0 +1,41 @@
+package http
+
+import (
+	"github.com/LerianStudio/midaz/common/mlog"
+	commonHTTP "github.com/LerianStudio/midaz/common/net/http"
+	"github.com/LerianStudio/midaz/components/transaction/internal/app/command"
+	"github.com/LerianStudio/midaz/components/transaction/internal/app/query"
+	a "github.com/LerianStudio/midaz/components/transaction/internal/domain/assetrate"
```

Comment:
please, use **ar** for **asset_rate** because we are using **a** for **accounts**, just to normaliza the imports.